app-ui/src/assets/main.css

```diff
@@ -1,11 +1,15 @@
 @import './base.css';
 
 #app {
-  max-width: 1280px;
+  /* max-width: 1280px; */
+  /* I recommend against setting max width. Better ways to do this*/
```

Comment:
Please go ahead and remove this, rather than commenting out.

There are also a few commented-out styles below. Go ahead and just delete them too.

Reply:
Will do